packages/mobile/package.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "@getstanza/mobile",
+  "version": "0.0.1-beta",
```

let's keep versions aligned. It makes releasing upgrades much easier
```suggestion
  "version": "0.0.7-beta",
```

---

Okay, I'll check this out.